.github/workflows/go.yml

```diff
@@ -19,10 +19,23 @@ jobs:
         go-version-file: "go.mod"
 
     - name: golangci-lint
-      uses: golangci/golangci-lint-action@v3
+      uses: smartcontractkit/golangci-lint-action@54ab6c5f11d66a92d14c3f7cc41ea13f676644bd
       with:
         # Optional: version of golangci-lint to use in form of v1.2 or v1.2.3 or `latest` to use the latest version
-        version: latest
+        version: v1.52.1
+        allow-extra-out-format-args: true
+        args: --out-format checkstyle:golangci-lint-report.xml
+
+    - name: Print golangci lint report
+      if: always()
+      run: test -f golangci-lint-report.xml && cat golangci-lint-report.xml || true
```

Comment:
Is the `|| true` needed so that this step always succeeds, whether the report file exists or not?

Reply:
Yes that is the idea. Again copied over from chainlink repo as it is there.